### ISSUE TITLE
Bug in `ai-publons-square` width?

### ISSUE THREAD

**Issue by g-oikonomou** [jpswalsh/academicons#70]
I think there may be something funny with the width of `ai-publons-square`, see how there is an increased gap to its right compared to all other gaps.

<img width="309" alt="screen shot 2018-01-13 at 16 47 16" src="https://user-images.githubusercontent.com/705207/34908099-77789cbe-f882-11e7-899c-915193f467dd.png">

This is what element inspection shows:

<img width="325" alt="screen shot 2018-01-13 at 16 53 28" src="https://user-images.githubusercontent.com/705207/34908093-614f810a-f882-11e7-991a-13ad42756229.png">

I am using:

```
<li><a class="navbar-social ai ai-publons-square ai-3x" title="..." href="..."></a></li>
```
(ditto for the remaining icons) so nothing too fancy. The `a` element for `ai-publons-square` ends up with slightly larger width than all other `a`s.

` ai-publons` seems to be behaving normally.

Thanks for the great iconset!

**Comment by jpswalsh** [comment]
Thanks for catching that. Should be fixed now (v1.8.2).

**Comment by g-oikonomou** [comment]
Yep, I can confirm this fix. Thanks!
